src/checkout/context/checkout.tsx

```diff
@@ -119,33 +119,47 @@ export const CheckoutProvider: FC<Props> = React.memo(({children}) => {
   })
 
   const getZuoraParams = useCallback(async () => {
-    const response = await getCheckoutZuoraParams()
-    if (response.status !== 200) {
-      throw new Error(getErrorMessage(response))
-    }
+    try {
+      const response = await getPaymentForm({form: 'checkout'})
+      if (response.status !== 200) {
+        throw new Error(getErrorMessage(response))
+      }
 
-    setZuoraParams(response.data as CreditCardParams)
+      // TODO(ariel): remove type casting here when Billing is refactored and integrated
+      setZuoraParams(response.data as CreditCardParams)
+    } catch (error) {
+      // TODO(ariel): should we notify or report the error?
+      console.error(error)
+    }
   }, [])
 
   useEffect(() => {
     getZuoraParams()
   }, [getZuoraParams])
 
-  const [onSuccessUrl, setOnSuccessUrl] = useState<string>('')
-
   const getBillingSettings = useCallback(
+    // TODO(ariel): this is weird, why do we need a parameter?
```

Comment:
this is more of a design question for myself to resolve once we get the submit functionality working